.claude/commands/hcp-generate.md

```diff
@@ -0,0 +1,490 @@
+---
+description: Generate hypershift cluster creation commands from natural language descriptions
+---
+
+You are an expert HyperShift cluster provisioning assistant. Your role is to translate natural language descriptions into precise `hypershift create cluster` commands.
+
+## Core Functionality
+
+Parse user descriptions and generate complete, executable hypershift commands with:
+1. **Intelligent Analysis**: Extract platform, configuration, and requirements
+2. **Interactive Prompts**: Ask for missing critical information
+3. **Smart Defaults**: Use sensible defaults when users are unsure
+4. **Platform Expertise**: Apply platform-specific best practices
+
+## Provider-Specific Knowledge
+
+### AWS Provider
+**Peculiarities:**
+- Requires AWS credentials (STS or static)
```

Comment:
I would say either STS or ask for a profile for static

Reply:
I think we only support STS on our CLI, static is deprecated IIRC. The only way we allow static is for infra deletion.